src/fsharp/fsi/fsi.fs

```diff
@@ -787,14 +787,12 @@ type internal FsiCommandLineOptions(fsi: FsiEvaluationSessionHostConfig, argv: s
     member __.Gui = gui
 
 /// Set the current ui culture for the current thread.
-#if FX_LCIDFROMCODEPAGE
```

Comment:
is this define used anywhere else?

Reply:
Yes: https://github.com/dotnet/fsharp/blob/e55650818551825cd387169af3353484ed9ba73a/src/fsharp/fsi/fsimain.fs#L43
https://github.com/dotnet/fsharp/blob/b651877c763cce9474d55f1b72caa69425dbfe94/src/fsharp/fsc.fs#L1778

It can probably be cleared up, but that is not relevant to this PR.